operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -384,6 +384,15 @@ rbac:
   create: true
   # For GKE/EKS/AKS use 'type: psp'. For OpenShift use 'type: scc'
   type: psp
+  # Default pod security context for mimir and cache components
+  # When using OpenShift, set
```

Comment:
stray line, remove - we should have a task based document in the documentation instead
```suggestion
```